stubs/urllib3/urllib3/util/retry.pyi

```diff
@@ -12,8 +12,15 @@ ResponseError = exceptions.ResponseError
 log: Any
 
 class Retry:
+    DEFAULT_ALLOWED_METHODS: frozenset[str]
+    RETRY_AFTER_STATUS_CODES: frozenset[int]
+    DEFAULT_REMOVE_HEADERS_ON_REDIRECT: frozenset[list[str]]
```

Comment:
```suggestion
    DEFAULT_REMOVE_HEADERS_ON_REDIRECT: frozenset[str]
```

Looks like a frozenset with a single element to me, the string "Authorization" 🙂 (It's actually impossible to have a frozenset of lists, since lists aren't hashable!) https://github.com/urllib3/urllib3/blob/33afb5c96c435ab7118b5f41e26f96c290fef55b/src/urllib3/util/retry.py#L192

Reply:
Yes good point @AlexWaygood! My bad.   I will fix it now.  Thank you